modules/tinymce/src/themes/silver/test/ts/browser/editor/menubar/EditorMenubarRenderTest.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { Css } from '@ephox/sugar';
+import { TinyHooks, TinyUiActions } from '@ephox/wrap-mcagar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import { tinymce } from 'tinymce/core/api/Tinymce';
+
+describe('browser.tinymce.themes.silver.editor.menubar.EditorMenubarRenderTest', () => {
+  const hook = TinyHooks.bddSetup<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    setup: (editor: Editor) => {
+      editor.on('ScriptsLoaded', () => {
+        const orgLoad = editor.ui.styleSheetLoader.load;
+        editor.ui.styleSheetLoader.load = (url) => {
+          return new Promise((resolve) => {
+            setTimeout(() => {
+              return orgLoad(url).then(resolve);
+            }, 100);
+          });
+        };
+      });
+    }
+  }, []);
+
+  tinymce.addI18n('en', {
```

Comment:
This should be set in before and unset or restored in an after so that we don't get other test failing here that looks for the `File` menu.

Reply:
I added a clean-up in `after` but putting this code in the before isn't working